### PR TITLE
Added json output options

### PR DESCRIPTION
**Pull request by thedavecollins**
I have done this is a bit of a hurry for an internal project.

I would love it to be included  in the main project.

Dave

### CODE REVIEW

File: src/pytest_benchmark/plugin.py [hunk 1]
```diff
@@ -15,6 +18,8 @@
 from .timers import compute_timer_precision
 from .timers import default_timer
 
+import newhooks
```
Comment (ionelmc) [inline src/pytest_benchmark/plugin.py]: What's this `newhooks` module?

**Comment by ionelmc** [comment]
There's just one minor nit with the coding style (there rest of the code doesn't use camelCaseNaming for variables).

I have to check if the json has all the necessary info for comparing runs (eg: redisplay previous result side-by-side)

**Comment by ionelmc** [comment]
Also, this fixes #2.

**Comment by thedrow** [comment]
I would really love to see this feature merged but the build is failing.

**Comment by ionelmc** [comment]
I've started to work on #2 from these commits in the master branch.

**Pull request closed without merging**